soc/xtensa/nxp_adsp/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # NXP i.MX/i.MX RT DSP family CMake file
 #
-# Copyright (c) 2021,2023 NXP
+# Copyright (c) 2021, 2023 NXP
```

Comment:
This kind of changes should be squashed back to the commit introducing these lines originally.